mne/decoding/regressor.py

```diff
@@ -0,0 +1,104 @@
+# Authors: Romain Trachel <trachelr@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from .base import BaseEstimator
+
+
+class LinearRegressor(BaseEstimator):
```

Comment:
The codes between classifiers and regressors are identical aren't they? If so just have a linear estimator object that checks whether the values in y are integers.